mods/cnc/maps/gdi09/gdi09-AI.lua

```diff
@@ -0,0 +1,201 @@
+--[[
+   Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+
+AttackPaths = { { waypoint7 }, { waypoint8 } }
+NodBase = { handofnod, nodairfield, nodrefinery, NodCYard, nodpower1, nodpower2, nodpower3, nodpower4, nodpower5, gun5, gun6, gun7, gun8, nodsilo1, nodsilo2, nodsilo3, nodsilo4, nodobelisk}
+
+PatrolProductionQueue = { }
+InfantryAttackGroup = { }
+InfantryGroupSize = 5
+InfantryProductionCooldown = DateTime.Minutes(3)
+InfantryProductionTypes = { "e1", "e1", "e1", "e3", "e3", "e4" }
+HarvesterProductionType = { "harv" }
+VehicleAttackGroup = { }
+VehicleGroupSize = 5
+VehicleProductionCooldown = DateTime.Minutes(3)
+VehicleProductionTypes = { "bggy", "bggy", "bggy", "ltnk", "ltnk", "arty" }
+StartingCash = 14000
+
+BaseRefinery = { type = "proc", pos = CPos.New(12, 25), cost = 1500 }
```

Comment:
Can we use `Actor.Cost("proc")` instead of hardcoding costs that may change?

Reply:
Good idea, but Imho we shouldn't waste much more time on this system anyway.

Reply:
Thanks for the hint. Could be changed in the other missions as well but I guess it will be replaced soon ayway?